include/osquery/distributed.h

```diff
@@ -194,7 +194,7 @@ class DistributedPlugin : public Plugin {
    */
   virtual Status writeResults(const std::string& json) = 0;
 
-  /// Main entrypoint for distirbuted plugin requests
+  /// Main entrypoint for distribute plugin requests
```

Comment:
I think this actually should be `distributed`

Reply:
good catch